Okta/OktaAppAuth.swift

```diff
@@ -75,11 +114,6 @@ public func getUser(_ callback: @escaping ([String:Any]?, OktaError?) -> Void) {
     _ = UserInfo(token: tokens?.accessToken) { response, error in callback(response, error) }
 }
 
-public func clear() {
```

Comment:
Since we're planning to remove the `clearTokens` method, please make sure this gets added back.

Reply:
Done.